### ISSUE TITLE
discuss: remove breadcrumb menu on the profile page

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/invenio-app-rdm#1383]
## Issue
The breadcrumb menu on the profile page does not have much purpose. The link to “Home” is only linking to the current page/section, the “Account” link is linking to the Profile section listed right below. I.e. the user is not taken to any different pages when attempting to navigate in the breadcrumbs.

All the information you need is already in the side-menu, and navigation-wise you’re only one level away from “Home” (`/`), not two as indicated in the breadcrumbs. Attempting to access `/account/` (where I’m expecting to get with the breadcrumbs menu) gives a 404. And the url even tells me that I’m actually 3 levels down from “Home” `/account/settings/security/`, where none of the preceding paths lead to anywhere (all 404).

## Screenshot
<img width="561" alt="Screenshot 2022-03-15 at 11 34 26" src="https://user-images.githubusercontent.com/21052053/158359260-a15546e4-8fae-4614-9384-e1045a698217.png">

## Suggestion
- Have redirects for `/account/` and `/account/.../` to the actual account page (`/account/settings/profile/`).
- Replace the breadcrumbs with a descriptive headline “My account” / “Manage my account”.
